hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/HazelcastSqlValidator.java

```diff
@@ -176,9 +176,10 @@ public SqlNode validate(SqlNode topNode) {
             return explainStatement;
         }
 
-        // Handling same corner case as with EXPLAIN
         if (topNode instanceof SqlAnalyzeStatement) {
             SqlAnalyzeStatement analyzeStatement = (SqlAnalyzeStatement) topNode;
+            // Note: we're using custom validate method to handle same issues as for EXPLAIN.
```

Comment:
This comment should be a line lower. Validation of the ANALYZE itself is AFAIU for validating the options and unrelated to corner cases